qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1007,6 +1007,12 @@ def run(
                 instance=qrt_options.instance, backend_name=qrt_options.backend
             )
             hgp_name = hgp.name
+        backend = self.backend(name=qrt_options.backend, instance=hgp_name)
+        status = backend.status()
+        if status.operational is True and status.status_msg != "active":
+            warnings.warn(
+                f"The backend {backend.name} is currently {status.status_msg}."
```

Comment:
```suggestion
                f"The backend {backend.name} currently has a status of {status.status_msg}."
```

sorry, small nit: `The backend {} currently has a status of internal/maintenance/dedicated` sounds better than 
`The backend {} is currently internal/maintenance/dedicated`